lib/discourse_data_explorer/report_generator.rb

```diff
@@ -73,6 +78,33 @@ def self.build_report_pms(query, table = "", targets = [], attach_csv: false, re
       pms
     end
 
+    def self.build_report_post(query, table = "", attach_csv: false, result: nil)
+      pms = []
+
+      upload = create_csv_upload(query, result) if attach_csv
+
+      post = {}
+      post["raw"] = "### Scheduled Report for #{query.name}\n\n" +
```

Comment:
this is a bit tricky, not localized at all, I guess it is just a caveat of the feature?

Reply:
Maybe we just remove almost all text except for a preamble - and translate that?

Reply:
Yes you're right, I copy-pasted here and didn't follow my own rules about I18n :grimacing: Will have a look at the state of things, maybe we can translate the whole thing...